backend/src/main/resources/application.properties

```diff
@@ -60,6 +60,7 @@ quarkus.datasource.jdbc.max-size=16
 quarkus.hibernate-orm.dialect=org.hibernate.dialect.PostgreSQL10Dialect
 quarkus.hibernate-orm.database.globally-quoted-identifiers=true
 quarkus.flyway.migrate-at-start=true
+quarkus.flyway.ignore-missing-migrations=true
```

Comment:
What's that for?

Reply:
Please don't ask me questions that I have no answer to. 😅

I probably just forgot to revert a local change that I needed for testing.

Reply:
This is a workaround for a hot reloading issue during `quarkus:dev`. See #212.

I've committed a workaround on develop, so this should no longer be a part of this PR. (diff removed in 57ce308)